### PR TITLE
weka clustering support for atoml

### PR DESCRIPTION
**Pull request by THaar50**

### CODE REVIEW

File: src/main/resources/weka-clustering-morphtest.template [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                expectedMorphedCluster = originalCluster;
+                expectedMorphCounts[(int) expectedMorphedCluster]++;
+                expectedMorphDistributions[i] = clusterer.distributionForInstance(data.instance(i))[(int) originalCluster];
+                expectedClustersMap.get((int) expectedMorphedCluster).add(i);
+                double morphedCluster = morphedClusterer.clusterInstance(data.instance(i));
+                morphedCounts[(int) morphedCluster]++;
+                morphedDistributions[i] = morphedClusterer.distributionForInstance(data.instance(i))[(int) morphedCluster];
+                morphedClustersMap.get((int) morphedCluster).add(i);
+
+                if (!(Double.compare(expectedMorphedCluster, morphedCluster) == 0)) {
```
Comment (sherbold) [inline src/main/resources/weka-clustering-morphtest.template]: This should be calculated with the "cluster matching" in mind.

**Review by sherbold — COMMENTED** [review]
The diff of the README covers the whole file. The changes only seem to be in the last section. Please check why this is.

**Comment by sherbold** [comment]
The PR looks good to me. Is there something missing or should I merge this?

**Comment by THaar50** [comment]
All files needed for Weka clustering are included in this PR now. You can merge it.

**Pull request merged**